### ISSUE TITLE
Defaults values in SAM open source appear to be set up wrong

### ISSUE THREAD

**Issue by adyreson** [NREL/SAM#228]
Using the latest versions of lk,wex, ssc, and sam builds for x64 current 10-7-19, I'm finding that CSP ST Molten Salt has error messages upon starting such as "fail: [cp_battery_nameplate] = f( en_batt,batt_computed_bank_capacity )
fail: [cp_battery_nameplate] = f( en_batt,batt_computed_bank_capacity )
fail: [cp_battery_nameplate] = f( en_batt,batt_computed_bank_capacity )
no variables calculated in a single iteration, cannot make progress!
eqn not evaluated: [cp_battery_nameplate] = f( en_batt,batt_computed_bank_capacity)"

In addition for CSP ST Molten salt as well as other technologies, I get errors about the default values:"Defaults file is likely out of date: MSPT_Single Owner.txt Variables: 0 loaded but not in configuration, 0 wrong type, defaults file has 732, config has 732. Would you like to update the defaults with the current values right now? (Otherwise press Shift-F10 later)" 
For most technologies if I try to hit Simulate after getting this message and accepting the current values as default, SAM closes.

**Comment by dguittet** [comment]
Thanks @adyreson for pointing out these bugs. We'll definitely have to fix these before the next release. 

I have a good idea where the first issue is coming from, and will take a look at the second.

**Comment by janinefreeman** [comment]
The second error shows up because the equations aren't working- if the equations are resolved, that second pop up should go away.

**Comment by dguittet** [comment]
@sjanzou @janinefreeman 

This is due to the new Financial Capacity Payments UI page that has been added to all single owner configurations. The following equation causes an error in all technologies without "en_batt" variable.

```
equations{ 'cp_battery_nameplate' } = define() {
	if ( ${en_batt}){
		return ${batt_computed_bank_capacity}/1000;
	}
	else 
		return 0;
};
```

I've changed this equation to first check if it's "Flat Plate PV" or if financing is "None". However, this page needs to be fixed to display the battery fields only if appropriate.
